### PR TITLE
adding works

### PR DESCRIPTION
**Pull request by OlafKozak**
first homework
albo nie nie mam lepszy opis
dupa

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @OlafKozak

**Comment by ziobron** [comment]
⏱ 3 XP extra for @OlafKozak for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,20 @@
 #include <string>
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:2]: Please never insert in Your commit message vulgarism or similar improper names.
Always describe what you commit contains.

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+        return std::to_string(first + second);
+    } 
+    else if(command == "subtract"){
+        return std::to_string(first - second);
+    }  
+    else if(command == "multiply"){
+        return std::to_string(first * second);
+    }  
+    else if(command == "divide"){
+        if(second == 0) return "Division by 0";
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:15]: One command per line

**Review by nauka-programowania-MA — APPROVED** [review]
Please never insert in Your commit message vulgarism or similar improper names.
Always describe what you commit contains.

**Pull request closed without merging**